pkgs/applications/version-management/srcml/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl, cmake, libxml2, libxslt, boost, libarchive, python, antlr }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  version = "0.9.5";
+  name = "srcml-${version}";
+
+  src = fetchurl {
+    url = "http://www.sdml.cs.kent.edu/lmcrs/srcML-src.tar.gz";
```

Comment:
I still would like to see either an versioned upstream url or a different source.